src/main/java/hudson/plugins/copyartifact/DownstreamBuildSelector.java

```diff
@@ -188,6 +188,11 @@ public FormValidation doCheckUpstreamProjectName(
             if (containsVariable(upstreamProjectName)) {
                 return FormValidation.ok();
             }
+
+            if (project == null) {

```

Comment:
While you are it, you should replace `AbstractProject` with `Job` to get Pipeline compatibility missed in #55. Ditto in other related validation methods. CC @apemberton: currently with Pipeline this throws a `NullPointerException`; fixing the PR as written would make it unconditionally return OK; also fixing the type of the parameter would make it actually perform the desired validation.